### PR TITLE
Add documentation about extra update ops to tf.layers.batch_normalization

### PR DESCRIPTION
**Pull request by mrahtz**
`tf.layers.batch_normalization` uses extra ops for updating its moving_mean and moving_variance variables, but these are added to the graph in such a way that they will not be a dependency of the main training op. Instead, they must be run separately, accessed through the `tf.GraphKeys.UPDATE_OPS` collection. This was explained in the documentation for `tf.contrib.layers.batch_norm`, but this information seems not to have survived the transition to `tf.layers.batch_normalization`.

Long term, if possible, it would be really nice if we didn't have to do anything extra to get these update operations to work. But in the meantime, here's a patch for the documentation which explains how to make sure the extra ops do get run.

### CODE REVIEW

**Comment by mrahtz** [comment]
I signed it!

File: tensorflow/python/layers/normalization.py [hunk 1]
```diff
@@ -267,6 +267,23 @@ def batch_normalization(inputs,
 
   Sergey Ioffe, Christian Szegedy
 
+  Note: the operations which update moving_mean and moving_variance will not be
```
Comment (fchollet) [inline tensorflow/python/layers/normalization.py]: Use "`" around code keywords.

**Review by fchollet — APPROVED** [review]

File: tensorflow/python/layers/normalization.py (hunk 1 above)
Comment (mrahtz) [inline tensorflow/python/layers/normalization.py]: Fixed in extra commit.

**Comment by caisq** [comment]
@tensorflow-jenkins test this please

**Comment by mrahtz** [comment]
Huh, so it failed on `//tensorflow/python:layers_normalization_test`. Let me run tests locally to see why it failed.

**Comment by caisq** [comment]
It's a known flaky test. Merging PR.
